aptos-move/framework/aptos-stdlib/sources/debug.move

```diff
@@ -1,6 +1,272 @@
 /// Module providing debug functionality.
 module aptos_std::debug {
-    native public fun print<T>(x: &T);
+    use std::string::String;
 
-    native public fun print_stack_trace();
+    public fun print<T>(x: &T) {
+        native_print(format(x));
+    }
+
+    public fun print_stack_trace() {
+        native_print(native_stack_trace());
+    }
+
+    inline fun format<T>(x: &T): String {
+        aptos_std::string_utils::debug_string(x)
+    }
+
+    native fun native_print(x: String);
+    native fun native_stack_trace(): String;
```

Comment:
this is a breaking change, we probably should have kept around the original behavior for some period of time and deprecated it...

cc: @gregnazario